mungegithub/github/github.go

```diff
@@ -726,40 +747,45 @@ func (obj *MungeObject) RemoveLabel(label string) error {
 // GetHeadAndBase returns the head SHA and the base ref, so that you can get
 // the base's sha in a second step. Purpose: if head and base SHA are the same
 // across two merge attempts, we don't need to rerun tests.
-func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool) {
+//CHECK-UPDATED
+func (obj *MungeObject) GetHeadAndBase() (headSHA, baseRef string, ok bool, err error) {
```

Comment:
If the caller has no reasonable actions that could be conditionalized on the error, then I think returning the error is mostly just verbosity.

In the case of this function, if it fails an optimization is missed. The caller isn't going to bother to retry, so there's not much point in expressing the difference between various categories of failures. If anything, you could add that as a comment here.